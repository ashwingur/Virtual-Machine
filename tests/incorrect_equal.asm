FUNC LABEL 0
    PRINT VAL 3
    EQU VAL 1
    RET