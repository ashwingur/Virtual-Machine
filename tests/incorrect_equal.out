3
The argument for EQU must be of register type.
